### ISSUE TITLE
Pip can't find spleeter

### ISSUE THREAD

**Issue by jhm0799** [deezer/spleeter#10]
I can't install this at all with pip. it always says:

ERROR: Could not find a version that satisfies the requirement spleeter (from versions: none)
ERROR: No matching distribution found for spleeter

whenever I type:

pip install spleeter

Any ideas? This is annoying...

Also, if it means much: I'm using Windows 10 version 1903

**Comment by Faylixe** [comment]
Could you give us more information ? In which context you are running `pip` ? (Command line, Python distribution, through `virtualenv` or `Conda`, into a container, etc ...).

The `spleeter` package is available on *PyPi*, so there is no chance that would happen.

**Comment by jhm0799** [comment]
CMD. I'm running it as administrator. I installed the normal Python release (3.8.0 64-bit)

**Comment by jhm0799** [comment]
Closing this issue. I decided to just install Anaconda and use it that way.

**Comment by batzlerg** [comment]
@jhm0799 I encountered the same issue and it's caused by your version of python being outside the spleeter reqs of `Python >=3.6, <3.8` as listed on its PyPI page (special emphasis on the `<3.8` part).

The error message pip gives is really unclear about this being the cause unless you run it with verbose output (`pip install spleeter -vvv`) to trace the steps it takes to resolve the install command. I installed 3.7.5 and ran the install again and everything worked as expected.
